Cql/Cql.Compiler/ExpressionBuilder.ComparisonOperators.cs

```diff
@@ -43,21 +42,15 @@ protected Expression Equal(Expression left, Expression right)
                 }
                 else if (right.Type == typeof(string))
                 {
-                    var call = BindCqlOperator(CqlOperator.EnumEqualsString,
-                        Expression.Convert(left, typeof(object)),
-                        right);
-                    return call;
+                    return _operatorBinding.BindToMethod(CqlOperator.EnumEqualsString, Expression.Convert(left, typeof(object)), right);
```

Comment:
These conversions should eventually be moved to your god binding method I think.

Reply:
I'll create a second issue for this. I'm considering merging the CqlOperatorBinding into ExpressionBuilder.

Reply:
Issue #297